BasicVariables.cs

```diff
@@ -0,0 +1,148 @@
+
+            floatVariable = 1.0F;
+            doubleVariable = 1.0;
+            Console.WriteLine($"Compared '1.0' value for Float and Double: {floatVariable == doubleVariable}");
+
+            floatVariable = 0.1F;
+            doubleVariable = 0.1;
+            Console.WriteLine($"Compared '0.1' value for Float and Double: {floatVariable == doubleVariable}");
+        }
+
+        /// <summary>
+        /// This method will always fail because of System.DivideByZeroException
+        /// </summary>
+        public void DivideValuesByZero()
+        {
+            intVariable = 20;
+            longVariable = 20;
+            doubleVariable = 20.0000;
+            floatVariable = 20.00F;
+
+            Console.WriteLine($"Divided Int by zero: {intVariable / 0}");
+            Console.WriteLine($"Divided Long by zero: {longVariable / 0}");
+            Console.WriteLine($"Divided Double by zero: {doubleVariable / 0}");
+            Console.WriteLine($"Divided Float by zero: {floatVariable / 0}");
+        }
+
+        /// <summary>
+        /// Divide value by int and assign the result to the variable
+        /// </summary>
+        public void DivideValuesByIntAndAssignResult()
+        {
+            intVariable = 22;
+            longVariable = 22;
+            doubleVariable = 22.0000;
+            floatVariable = 22.00F;
+
+            intVariable /= 3;
+            Console.WriteLine($"Divided Int by '3' and assigned result: {intVariable}");
+
+            longVariable /= 3;
+            Console.WriteLine($"Divided Long by '3' and assigned result: {longVariable}");
+
+            doubleVariable /= 3;
+            Console.WriteLine($"Divided Double by '3' and assigned result: {doubleVariable}");
+
+            floatVariable /= 3;
+            Console.WriteLine($"Divided Float by '3' and assigned result: {floatVariable}");
+        }
+
+        /// <summary>
+        /// Divide value by double, assign the result to the variable and make Round operation
+        /// </summary>
+        public void DivideValuesByDoubleAndMakeRound()
+        {
+            intVariable = 25;
+            longVariable = 25;
+            doubleVariable = 25.0000;
+            floatVariable = 25.00F;
+
+            double myDouble1 = intVariable;
+            myDouble1 /= 3.0;
+            Console.WriteLine($"Divided Int by '3' and assigned result: {Math.Round(myDouble1)}");
+
+            double myDouble2 = longVariable;
+            myDouble2 /= 3.0;
+            Console.WriteLine($"Divided Long by '3' and assigned result: {Math.Round(myDouble2)}");
+
+            doubleVariable /= 3.0;
+            Console.WriteLine($"Divided Double by '3' and assigned result: {Math.Round(doubleVariable)}");
+
+            double myDouble3 = floatVariable;
+            myDouble3 /= 3.0;
+            Console.WriteLine($"Divided Float by '3' and assigned result: {Math.Round(myDouble3)}");
+        }
+
+
```

Comment:
try also comparison of all values in the cycle, e.g.:

double[] doubleArray = { 0.5, 0.7, 1.0, 0.1 };
float[] floatArray = { 0.5f, 0.7f, 1.0f, 0.1f };

for (int i = 0; i < doubleArray.Length; i++)
{
 System.Diagnostics.Debug.WriteLine(doubleArray[i] + " == " + floatArray[i] + ": " + (doubleArray[i] == floatArray[i]));
 }